src/librustc_codegen_ironox/type_.rs

```diff
@@ -247,26 +247,16 @@ impl BaseTypeMethods<'tcx> for CodegenCx<'ll, 'tcx> {
     fn val_ty(&self, v: Value) -> Type {
         let module = &self.module.borrow();
         match v {
-            Value::ConstUint(const_idx) => {
```

Comment:
This hunk is just re-formatting I think?

Reply:
It is.